examples/style_sheets/style_sheets_reference.py

```diff
@@ -65,6 +66,12 @@ def plot_colored_circles(ax, prng, nb_samples=15):
     for sty_dict, j in zip(plt.rcParams['axes.prop_cycle'], range(nb_samples)):
         ax.add_patch(plt.Circle(prng.normal(scale=3, size=2),
                                 radius=1.0, color=sty_dict['color']))
+    ax.grid(visible=True)
+    # Add annotation for enabling grid
+    ax.annotate('ax.grid(True)', xy=(1, 1),
```

Comment:
This looks good, but why did you use annotate here, versus `title`?  Might also consider `family='monospace'`?

Reply:
This was added by the last author and I just incorporated those changes. 

Can refactor to use title and change the font family if desired. 

Let me know @jklymak

Reply:
@jklymak ended up implementing your suggestions. Changed to `title` and `family='monospace'`. Fewer lines of code and more readable. Thanks.